docs/ClusterSetup.md

```diff
@@ -221,6 +224,7 @@ zookeeperServers=zk1.us-west.example.com:2181,zk2.us-west.example.com:2181,zk3.u
 globalZookeeperServers=zk1.us-west.example.com:2184,zk2.us-west.example.com:2184,zk3.us-west.example.com:2184
 
 clusterName=us-west
+statusFilePath=broker-status-file
```

Comment:
Maybe worth changing the default here? https://github.com/yahoo/pulsar/blob/master/conf/broker.conf#L78